sources/packages/forms/src/form-definitions/sfaa2021-22.json

```diff
@@ -16604,6 +16694,85 @@
                     "input": false,
                     "tableView": false,
                     "components": [
+                        {
+                            "label": "I have been authorized by my parent(s) to submit their financial information including their net income (previous tax year's return's Line 15000 information)",
+                            "labelWidth": "",
+                            "labelMargin": "",
+                            "description": "",
+                            "tooltip": "",
+                            "shortcut": "",
+                            "inputType": "checkbox",
+                            "customClass": "",
+                            "tabindex": "",
+                            "hidden": false,
+                            "hideLabel": false,
+                            "autofocus": false,
+                            "disabled": false,
+                            "tableView": false,
+                            "modalEdit": false,
+                            "persistent": true,
+                            "protected": false,
+                            "dbIndex": false,
+                            "encrypted": false,
+                            "redrawOn": "",
+                            "clearOnHide": true,
+                            "customDefaultValue": "",
+                            "calculateValue": "",
+                            "calculateServer": false,
+                            "allowCalculateOverride": false,
+                            "validate": {
+                                "required": true,
+                                "customMessage": "",
+                                "custom": "",
+                                "customPrivate": false,
+                                "json": "",
+                                "strictDateValidation": false,
+                                "multiple": false,
+                                "unique": false
+                            },
+                            "errorLabel": "",
+                            "errors": "",
+                            "key": "iHaveBeenAuthorizedByMyParentSToSubmitTheirFinancialInformationIncludingTheirNetIncomePreviousTaxYearsReturnsLine15000Information",
```

Comment:
Even if this is not participating in the workflow or processed in the API it would be good to give a reasonable variable name.